test/scripts/common.inc.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Shell include files should neither have a shebang nor should they have executable permissions. I would also name the files `.inc` instead of `.inc.sh` to indicate that they are include files.

Reply:
Thanks mate! I remove the shebang and rename them; about executable permissions, they include files does not have executable permissions.